src/Layout/Presentation/SingleParty/PartySelect.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+export interface PartySelectProps {
+    selectParty: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+    partyCodeList: string[];
+    partyMap: _.Dictionary<string>;
+    partySelected: string;
+}
+
+export class PartySelect extends React.Component<PartySelectProps> {
+    render() {
+        return (
+            <div className="field">
+                <div className="control">
+                    <div className="select is-fullwidth is-primary is-medium">
+                        <select
+                            id="district"
+                            onChange={this.props.selectParty}
+                            value={this.props.partySelected}
```

Comment:
For future reference, this can probably be refactored into a "SingleView" -- I suspect this part of the SingleDistrict and SingleParty components are pretty much the same/behave the same.

Reply:
Probably, would take quite a lot of reworking, so I will not attempt that right now.